### PR TITLE
Make sure we load cookie list from OneTrust on /cookies page

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
On /hedvig/cookies page, the list of cookies along with descriptions is fetched via the OneTrsut script. Make sure we load the script on page load to display the content and make sure user an update cookie preferences 

Screenshot 2024-05-13 at 10 58 40

## Justify why they are needed
Ensure we comply with GDPR

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4251** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4251?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/blocks/CookieListBlock.tsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+import { loadOneTrustAtom } from '@/components/CookieConsent/OneTrustAtom'
 import * as GridLayout from '@/components/GridLayout/GridLayout'
 
 export const CookieListBlock = () => {
+  const { t } = useTranslation('cookieConsent')
+  const setLoadOneTrust = useSetAtom(loadOneTrustAtom)
+
+  // Load OneTrust script to display the cookie list and handle user preferences
+  useEffect(() => {
+    setLoadOneTrust(true)
```
Comment (Youakeem) [inline apps/store/src/blocks/CookieListBlock.tsx:16]: Is loading the one trust script a sync or an async operation. We we need to wait for it to be loaded before allowing the user to make updates?

Comment (gustaveen) [reply apps/store/src/blocks/CookieListBlock.tsx:16]: The button to update settings will be unresponsive before the script is loaded so we don't need to handle that case I'd say

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/blocks/CookieListBlock.tsx (hunk 1 above)
Comment (Youakeem) [reply apps/store/src/blocks/CookieListBlock.tsx:16]: Alright, sounds good!

**Review by Youakeem — APPROVED** [review]

**Comment by gustaveen** [comment]
### Merge activity

* **May 13, 10:42 AM EDT**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4251).

**Pull request merged**